pkg/apis/v1alpha1/common.go

```diff
@@ -84,6 +84,11 @@ type ObjectReference struct {
 	APIVersion string `json:"apiVersion,omitempty"`
 }
 
+type NameAndNamespace struct {
```

Comment:
I think it would be more intent revealing if it were named `ServiceAccountRef[erence]`

Reply:
I'm definitely open to renaming that. I think it might make sense to keep it generic because the only fields on there are `name` and `namespace` which are not specific to service accounts.

We already have an `ObjectReference` type which has kind, apiversion, name, namespace. My first thought was we could call this new type `SimpleObjectReference` or something to that effect.

What do you think?

Reply:
The reason I brought this up was that I was drawing parallels with `ClusterTemplateReference` and `DeliveryClusterTemplateReference`. Additionally, I considered design of core k8s types, say `ClusterRoleBinding`  -- you'll see the role binding has a very specific type for the ref `RoleRef` even though this has very generic fields like `APIGroup` , `Kind` and `Name`.

Reply:
Interesting. I think the ClusterTemplateRef is different because we’re doing validation on those to ensure that only certain kinds are used. 

But the RoleRef seems similar enough that I’m convinced we can change to ServiceAccountRef. If we ever have a need to reuse the type we can think about renaming then